### PR TITLE
Tentatively re-enable WS-Security tests that were previously disabled…

### PR DESCRIPTION
**Pull request by asoldano**
…. I believe the tests could be transitively failing because of WFLY-5067; moreover I've applied few changes to isolate test CXF busses and explicitly set a dependency to the jbossws-cxf-client module on server side (reccomended approach).

Please let this PR be tested multiple times before eventually merging.

### CODE REVIEW

**Comment by asoldano** [comment]
... at least it looks like the tests are constantly failing now, not transitively. I'm going on trying isolating the problem by committing tentantive changes here, as the failures are not reproducible at all locally here...

File: testsuite/integration/ws/src/test/java/org/jboss/as/test/integration/ws/wsrm/ReliableCheckHandler.java [hunk 1]
```diff
@@ -63,50 +69,52 @@ private synchronized void handleContext(SOAPMessageContext smc) {
...
         SOAPMessage message = smc.getMessage();
 
-        invoked++;
-
         if (outboundProperty.booleanValue()) {
-            log.debug("Outgoing message:");
+            log.info("Outgoing message:");
         } else {
-            log.debug("Incoming message:");
+            log.info("Incoming message:");
```
Comment (ctomc) [inline testsuite/integration/ws/src/test/java/org/jboss/as/test/integration/ws/wsrm/ReliableCheckHandler.java]: this should probably be log.isTraceEnabled() no need to fill up logs

File: testsuite/integration/ws/src/test/java/org/jboss/as/test/integration/ws/wsrm/ReliableCheckHandler.java [hunk 2]
```diff
@@ -63,50 +69,52 @@ private synchronized void handleContext(SOAPMessageContext smc) {
...
             message.writeTo(os);
             os.flush();
-            log.debug("");
+            log.info("");
         } catch (Exception e) {
-            log.debug("Exception in handler: " + e);
+            log.info("Exception in handler: " + e);
         }
-        log.debug("-----------");
+        log.info("-----------");
```
Comment (ctomc) [inline testsuite/integration/ws/src/test/java/org/jboss/as/test/integration/ws/wsrm/ReliableCheckHandler.java]: similary here, we should probably just keep debug, info was needed for debugging

File: testsuite/integration/ws/src/test/resources/jboss-cli-logging.properties [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Additional logger names to configure (root logger is always configured)
```
Comment (ctomc) [inline testsuite/integration/ws/src/test/resources/jboss-cli-logging.properties]: you probably don't need this file as its only needed for cli tests.

**Pull request merged**